crates/ironrdp-server/src/acceptor/mod.rs

```diff
@@ -0,0 +1,99 @@
+use std::io;
```

Comment:
Just wondering: could this be in its own separate `ironrdp-acceptor` crate?

Reply:
Sure, I could see this as well

Reply:
What about the state machine driver? I noticed for the `ironrdp-connector`, that logic is extracted into `ironrdp_async::connector`.

For my case, the driver is more coupled to `tokio` (which is fine since the server will also always run on `tokio`), but that makes it a little harder to extract into `ironrdp-async` since that is runtime independent.
I will leave that logic in the acceptor crate for now, but I could refactor it a bit if you think is should belong to `ironrdp-async`.

Reply:
No problem, let’s keep this in ironrdp-acceptor for now. I don’t think everything should go inside ironrdp-async, I’ll think it over more later